build-system/global-configs/canary-config.json

```diff
@@ -14,18 +14,14 @@
     "twitter-default-placeholder-fade",
     "twitter-default-placeholder-pulse"
   ],
-
   "canary": 1,
```

Comment:
Please keep this whitespace.

Reply:
I think these whitespaces were killed by `gulp prettier` since I am too lazy to manually do such a thing, but I will gladly battle the prettier to unprettify the whitespace. ;)